client/src/contexts/QA.jsx

```diff
@@ -0,0 +1,166 @@
+import React, { useState, useContext } from 'react';
+import axios from 'axios';
+
+export const QAContext = React.createContext();
+
+export const QAProvider({ children }) {
+  const fetchAllQuestions = (product_id, page, count, callback) => {
+    const fetchParams = {
+      product_id,
+      page,
+      count
+    }
+
+    axios.get('/qa/questions', fetchParams)
+    .then((response) => {
+      callback(response)
+    })
+    .catch((err) => {
+      console.log('Server failed to collect Questions list');
+    })
+  }
+
+  const fetchAnswers = (question_id, callback) => {
+    const fetchParams = {
+      page,
+      count
+    }
+
+    axios.get(`/qa/questions/${questionID}/answers`, fetchParams)
+    .then((response) => {
```

Comment:
since we're adding `useState`, in line 31 we'll add `setAnswers(response);` so the fetched answers are added to state

Reply:
also, can indent `.then` block & `.catch` block